ethereum/block/testdata/generate-bindings.sh

```diff
@@ -1,9 +1,9 @@
-#!/bin/bash
+#!/usr/bin/env bash
 # Generate the UTContract.sol Go bindings into a file named ut_contract_test.go of the
 # parent directory for use by unit tests.
 
 # Use the contract's directory (where this script is) as the current working directory:
-cd "$(dirname "$(readlink -f "$0")")"
+cd "$(dirname "$(realpath "$0")")"
```

Comment:
These commands do the same thing, but `readlink` on macOS doesn't support the `-f` flag.  `realpath` is installed by default in Linux distributions, and by brew/coreutils on macOS.